core/synch.c

```diff
@@ -1827,11 +1826,11 @@ send_all_other_threads_native(void)
 #endif
 
     /* Suspend all threads except those trying to synch with us */
-    DEBUG_DECLARE(ok =)
-        synch_with_all_threads(desired_state, &threads, &num_threads,
+    bool ok = synch_with_all_threads(desired_state, &threads, &num_threads,
```

Comment:
Indentation of the 2 lines below is now off

Reply:
Done